### ISSUE TITLE
[Voiceover] VO button missing.

### ISSUE THREAD

**Issue by coreycurious** [authorly/interapptive-web#1077]
![screen shot 2013-11-05 at 8 37 44 am](https://f.cloud.github.com/assets/1136270/1475207/f5095822-4638-11e3-8bbc-84f081eb05c6.png)
